pkg/k8s/client/config.go

```diff
@@ -0,0 +1,56 @@
+package client
+
+import (
+	"os"
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/pkg/defaults"
+	"github.com/cilium/cilium/pkg/option"
+)
+
+type Config struct {
+	// K8sAPIServer is the kubernetes api address server (for https use --k8s-kubeconfig-path instead)
+	K8sAPIServer string
+
+	// K8sKubeConfigPath is the absolute path of the kubernetes kubeconfig file
+	K8sKubeConfigPath string
+
+	// K8sClientQPSLimit is the queries per second limit for the K8s client. Defaults to k8s client defaults.
+	K8sClientQPS float32
```

Comment:
I think this should be `K8sClientQPSLimit`.

Reply:
The flag is `--k8s-client-qps` and hence why the struct field reflects that (otherwise we'd need the `mapstructure:"k8s-client-qps"` struct tag). I just forgot to update the comment when I moved it from `pkg/option/config.go`. Fixed.